gradle-scripts/plugins.gradle

```diff
@@ -4,11 +4,10 @@ apply plugin: 'pmd'
 apply plugin: 'jacoco'
 
 /**
```

Comment:
The comments between 7 to 13 look unnecessary right now, can you remove them, please?

Reply:
e7a866946